src/main/java/at/hannibal2/skyhanni/data/jsonobjects/other/SkyblockItemsDataJson.kt

```diff
@@ -11,4 +11,5 @@ data class SkyblockItemData(
     @Expose val id: String?,
     @Expose @SerializedName("npc_sell_price") val npcPrice: Double?,
     @Expose @SerializedName("motes_sell_price") val motesPrice: Double?,
+    @Expose @SerializedName("stats") val stats: Map<String, Int>?,
```

Comment:
```suggestion
    @Expose @SerializedName("stats") val stats: Map<SkyblockStat, Int>?,
```

we already have a SkyblockStat enum, why are we dealing with strings here. As long as hypixel uses the same lowercase strings as neu does this should work without having to write any more logic into skyhanni

Reply:
Is this an enum or a class? If it is an enum then we have problems when hyoixel adds new stats

Reply:
Also why use serialised name when it is the same name?

Reply:
> Also why use serialised name when it is the same name?

I didnt even notice that but yes, thats not needed

Reply:
> Is this an enum or a class? If it is an enum then we have problems when hyoixel adds new stats

yeah i guess this is a problem. maybe best to leave as string here and convert to enum somewhere else. This way when working with item stats in functions we can use the enum instead of string comparisons, but then for something like showing stats on the tooltip we can fallback to the string (and maybe put something saying unknown stat report on discord so we can update/fix the list)

Reply:
Also why use serialised name when it is the same name?,es

Reply:
Firefox is loving me today